src/rust/ide/lib/utils/src/test.rs

```diff
@@ -97,3 +97,24 @@ for Collection {
         (v1,v2,v3,v4,v5)
     }
 }
+
+#[allow(clippy::type_complexity)]
+impl<Collection: IntoIterator> ExpectTuple
```

Comment:
Wouldnt it be betrter to just rename `Collection` to `T` ? It is common to use just `T` for generics and it will make the signature much easier to read. I spend now several seconds to count the number of items here,